src/libraries/Microsoft.Win32.SystemEvents/tests/ShutdownTest.cs

```diff
@@ -2,32 +2,45 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
-using System.Threading;
 using Microsoft.DotNet.RemoteExecutor;
 using Xunit;
 using static Interop;
 
-namespace Microsoft.Win32.SystemEventsTests
+namespace Microsoft.Win32.SystemEventsTests;
+
+public class ShutdownTest : SystemEventsTest
 {
-    public abstract class ShutdownTest : SystemEventsTest
```

Comment:
I'm not sure why this was originally marked as `abstract` but it seems that was causing the tests not to run.